__tests__/app.test.js

```diff
@@ -182,16 +186,80 @@ describe("POST /api/articles/:article_id/comments", () => {
         expect(msg).toBe("Bad request");
       });
   });
-  it("should respond with 400 bad request if article_id is a number but not present in the database", () => {
+  it("should respond with 404 bad request if article_id is a number but not present in the database", () => {
     return request(app)
       .post("/api/articles/0/comments")
       .send({
         username: "butter_bridge",
         body: "The most amazing comment ever",
       })
+      .expect(404)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("Not found");
+      });
+  });
+});
+
+describe("PATCH /api/articles/:article_id", () => {
+  it("should update an article by article_id", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send({ inc_votes: 3 })
+      .expect(200)
+      .then(({ body: { article } }) => {
+        expect(article).toMatchObject({
+          article_id: 1,
+          title: expect.any(String),
+          topic: expect.any(String),
+          author: expect.any(String),
+          body: expect.any(String),
+          created_at: expect.any(String),
+          votes: 103,
+          article_img_url: expect.any(String),
+        });
+      });
+  });
```

Comment:
Nice, but the test description could be more descriptive. What is actually happening here? what is being updated?